src/spec.c

```diff
@@ -164,8 +177,8 @@ const FieldSpec *IndexSpec_GetFieldBySortingIndex(const IndexSpec *sp, uint16_t
 const char *IndexSpec_GetFieldNameByBit(const IndexSpec *sp, t_fieldMask id) {
   for (int i = 0; i < sp->numFields; i++) {
     if (FIELD_BIT(&sp->fields[i]) == id && FIELD_IS(&sp->fields[i], INDEXFLD_T_FULLTEXT) &&
-        FieldSpec_IsIndexable(&sp->fields[i])) {
-      return sp->fields[i].name;
+      FieldSpec_IsIndexable(&sp->fields[i])) {
+      return HiddenName_GetUnsafe(sp->fields[i].fieldName, NULL);
```

Comment:
Since this function is used a lot (`getUnsafe()`), and small - consider making it `inline`.

Reply:
It is my understanding that an inline function has to be defined in the header file.
Since we don't know the actual structure of HiddenName and don't want to expose it in the header file then I can't make the GetUnsafe function an inline function.

Reply:
I think we can have an inline function in the .c file.

Reply:
I think in the end we said we can't do it without exposing the hidden name internals.